CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.16)
 
 set(VERSION_MAJOR "0")
-set(VERSION_MINOR "6")
-set(VERSION_PATCH "6")
+set(VERSION_MINOR "7")
+set(VERSION_PATCH "0")
```

Comment:
Approved, but not sure how to get this into the release that already happened ..

Reply:
0.7.1 when it happens.